src/test/java/com/dreamsportslabs/guardian/it/RefreshTokenIT.java

```diff
@@ -61,6 +60,10 @@ public void testValidRefreshToken() {
 
     assertThat(
         claims.get(JWT_CLAIM_RFT_ID), equalTo(DigestUtils.md5Hex(refreshToken).toUpperCase()));
+    assertThat(response.getCookies().containsKey("GUARDIAN-AT"), is(true));
+    assertThat(response.getCookie("GUARDIAN-AT"), isA(String.class));
+    // Validate that the access token is set in the cookie
+    assertThat(accessToken, equalTo(response.getCookie("GUARDIAN-AT")));
```

Comment:
String class assertion may not be required considering we are comparing with at.